assembl/tests/views_tests/test_graphql.py

```diff
@@ -177,6 +185,81 @@ def test_mutation_create_thematic_multilang_explicit_fr(graphql_request):
     }}}
 
 
+def test_mutation_create_thematic_upload_file(graphql_request):
+    # create thematic
+    import os
+    from io import BytesIO
+
+    class FileUpload(object):
+        file = BytesIO(os.urandom(16))
+        filename = 'path/to/img.png'
+
+    graphql_request.POST['variables.img'] = FileUpload()
+    res = schema.execute(u"""
+mutation myFirstMutation($img:String) {
+    createThematic(titleEntries:[
+        {value:"Comprendre les dynamiques et les enjeux", localeCode:"fr"},
+        {value:"Understanding the dynamics and issues", localeCode:"en"}
+    ],
+        identifier:"survey",
+        image:$img
+    ) {
+        thematic {
+            id,
+            title(lang:"fr"),
+            identifier,
+            imgUrl
+        }
+    }
+}
+""", context_value=graphql_request, variable_values={"img": u"variables.img"})
+    # The test doesn't use the same discussion id (sometimes it's 1, sometimes 8)
+    # depending on which tests are executed...
```

Comment:
Very, very possible, that some fixtures are not `function` scope. As a result, when the test is run, the test database creates several other discussions already...